src/kv/RocksDBStore.cc

```diff
@@ -242,6 +242,9 @@ int RocksDBStore::do_open(ostream &out, bool create_if_missing)
       return -EINVAL;
     }
   }
+
+  opt.statistics = rocksdb::CreateDBStatistics();
```

Comment:
Suggest to make statistics collection an optional feature,enabled by a config parameter. RocksDB manual has a mention that it might impact a performance a bit